### ISSUE TITLE
Regression: whole tree is hidden

### ISSUE THREAD

**Issue by PhiLhoSoft** [leny/atom-tree-ignore#18]
You introduced a regression when you removed the `_bUpdate` variable.
You replaced it in `_fUpdate` with `bState`, but you forgot several call sites: in `_fTreeViewHasMutated`, `_fProjectChangedPaths` and `_fAddAtomIgnoreFiles`.
There is also a strange bind for `"tree-view:toggle"`.

As a result, `_fUpdate` is called with `undefined` and so all the items are hidden...

I suggest to read the state at the start of the function if `bState` is undefined or null..

**Comment by PhiLhoSoft** [comment]
Tested with

```
if ( bState == null ) {
    bState = atom.config.get( "tree-ignore.enabled" );
}
```

with success, so far.

I would put this string, used everywhere, in a constant. And perhaps even wrap the call to config.get in a function, too. Just my taste, here, though.

**Comment by leny** [comment]
Ah, _shoot_! I'll correct that in a moment.

**Comment by leny** [comment]
Fixed in `0.6.1`.
